lib/associations/belongs-to-many.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var _ = require('lodash');
+
+module.exports = function(Resource, resource, association) {
+  // access points
+  var subResourceName =
+    association.target.options.name.plural.toLowerCase();
+
+  // Find reverse association
+  var associationPaired;
```

Comment:
a nitpicky comment: but shouldn't this just be called "reverseAssociation", and subsequently referred to as such in the thrown Error below?